docs/main/02-intro.md

```diff
@@ -1,71 +1,131 @@
 ---
 title: Docs Introduction
-description: Introduction to Kratix and how to get started quickly. Includes guides on how to read Kratix docs and pointers to other pages.
+description: Introduction to Kratix.
 keywords: [kratix documentation]
 sidebar_label: Introduction
 slug: /
 ---
 
-# Introduction
-
-Kratix is the open-source Platform Framework, that enables you to provide anything as-a-Service, while incorporating your key business processes and managing everything as a fleet.
-
 :::info
-Ready to get started? Follow our [Quick Start](01-quick-start.md) to jump right in.
+Looking for an **Enterprise solution**? Check out [Syntasso Kratix
+Enterprise (SKE)](https://syntasso.io/pricing)
 :::
```

Comment:
IMO, this is best suited under the introduction section as its like, "here is what OSS is...want enterprise?"

Reply:
pending more feedback from rest of CA in slack but I agree!

Reply:
moved it for now